app/views/course/assessment/submission/submissions/reload_answer.js.erb

```diff
@@ -5,3 +5,9 @@ $('<%= "#answer_#{@answer.id}"%>').replaceWith(
 <% if @assessment.guided? && guided_next_unanswered_question.nil? %>
   $('input.finalise').show();
 <% end %>
+
+$('.course-assessment-submission-submissions.edit .expected').shorten({
```

Comment:
Can directly use `$('.expected')` here, because this partial only get called in submission page. But it's OK to leave it as now.